src/components/main/index.tsx

```diff
@@ -5,7 +5,7 @@ import { clearLocalStorage, getLocalStorage, urlFormat } from "@/utils";
 import { Pages } from "@/pages";
 import { logoutRequest } from "@/services";
 
-const Main = () => {
+const UserActions = () => {
```

Comment:
Is it right the file name `src/components/main/index.tsx` ? I'm a bit confused about the name of the component `UserActions` and the file name `components/main`

Reply:
The idea is to control latter on the edit/logout here. But I can adjust it for now and then change it accordingly to the needs: 

<img width="308" alt="image" src="https://user-images.githubusercontent.com/116195115/200405428-5359da0b-8423-443f-a9ee-f2396e1df084.png">